setup.cfg

```diff
@@ -24,6 +24,9 @@ markers =
 # Disable internet access for tests not marked remote_data
 remote_data_strict = True
 
+[pytest]
+timeout = 300
```

Comment:
what does this do?!

Reply:
With the power of pytest-timeout, it kill the test after a set time of 300 seconds. Hopefully this means that it wont hang on travis like it is doing right now.

Reply:
a) does it error if you don't have it installed
b) does it fail the test after the timeout?

Reply:
a) Not a clue, I assume nothing happens since you need the plugin installed for the setting to count. 

b) Yes like so:
E       Failed: Timeout >30.0s

Reply:
I don't want to introduce another required test dep, but I don't think this does. Otherwise :+1:

Reply:
DONT USE NANO

Reply:
It didn't do anything on travis